zendro_cli.md

```diff
@@ -152,6 +152,36 @@ If a user has new data model definitions, it is convinient to use Zendro CLI for
 2. in `graphql-server` folder, it is possible to perform new generated migrations, which are generated after the last executed migration, by executing `zendro migration:up`. After that, the last executed migration and migration log are updated.
 3. in `graphql-server` folder, the last executed migration can be dropped by executing `zendro migration:down`. This will update the latest successful migration and add the dropped operation to the migration log. If there are some remaining records and associations in the table, by default an error is thrown. To forcefully drop the table, in spite of remaining records, set the environment variable `DOWN_MIGRATION` to `true` in `/graphql-server/.env` file and re-execute this down-migration.
 
+Note: for all `up` and `down` functions, there is a default argument called `zendro`. And it provides the access to different APIs in zendro layers (resolvers, models, adapters) and enables graphql queries. In model and adapter levels zendro can also access the storage handler which can interact with corresponding database management system. And there are some examples for model `movie` and adapter `dist_movie_instance1`.
```

Comment:
Note: for all `up` and `down` functions, there is a default argument called `zendro`. It provides the access to different APIs in zendro layers (resolvers, models, adapters) and enables graphql queries. In model and adapter levels zendro can also access the storage handler which can interact with corresponding database management system. Following are some examples for model `movie` and adapter `dist_movie_instance1`.